app/__init__.py

```diff
@@ -0,0 +1,29 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from config import config
+from flask_jwt import JWT
+
+db = SQLAlchemy()
+
+
+def create_app(config_name):
+    app = Flask(__name__)
+    app.config.from_object(config[config_name])
+    config[config_name].init_app(app)
+
+    jwt = JWT(app, authenticate, identity)
+
+    db.init_app(app)
+    jwt.init_app(app)
+
+    from .auth.views import auth
```

Comment:
Consider imports in the format `app.auth.views` , they are cleaner and only use imports within your code if you are trying to avoid cyclic imports otherwise put all imports at the top.